src/pfb/importers/gen3dict.py

```diff
@@ -275,7 +275,10 @@ def _array_type(property_type):
         if "description" in property_type:
             enum["name"] = property_type["description"]
         else:
-            enum["name"] = property_type["termDef"][0]["term"]
+            enum["name"] = property_type["term"]["termDef"]["term"]
```

Comment:
I think this block can be rewritten as
```python
enum["name"] = (
    property_type.get("description") or 
    property_type["term"]["termDef"]["term"] or 
    property_type["termDef"][0]["term"])
```

Reply:
yeah I was suppose to write this using .get() somehow I just forgot 🤦